definitions/npm/webpack_v5.x.x/flow_v0.104.x-/webpack_v5.x.x.js

```diff
@@ -678,6 +701,95 @@ declare module 'webpack' {
     >;
   }
 
+  declare type ProgressHandler = (percentage: number, msg: string, ...args: string[]) => void;
+
+  /**
+   * Options object for the ProgressPlugin.
+   */
+  declare type ProgressPluginOptions = {|
+    /**
+     * Show active modules count and one active module in progress message.
+     */
+    activeModules?: boolean,
+
+    /**
+     * Show dependencies count in progress message.
+     */
+    dependencies?: boolean,
+
+    /**
+     * Minimum dependencies count to start with. For better progress calculation. Default: 10000.
+     */
+    dependenciesCount?: number,
+
+    /**
+     * Show entries count in progress message.
+     */
+    entries?: boolean,
+
+    /**
+     * Function that executes for every progress step.
+     */
+    handler?: ProgressHandler,
+
+    /**
+     * Show modules count in progress message.
+     */
+    modules?: boolean,
+
+    /**
+     * Minimum modules count to start with. For better progress calculation. Default: 5000.
+     */
+    modulesCount?: number,
+
+    /**
+     * Collect percent algorithm. By default it calculates by a median from modules, entries and dependencies percent.
+     */
+    percentBy?: null | "entries" | "modules" | "dependencies",
+
+    /**
+     * Collect profile data for progress steps. Default: false.
+     */
+    profile?: null | boolean,
+  |};
+
+  declare type ProgressPluginArgument =
+    | ProgressPluginOptions
+    | ProgressHandler;
+
+  declare class ProgressPlugin {
+    constructor(options?: ProgressPluginArgument): $ElementType<
```

Comment:
`$ElementType`, `$PropertyType` are deprecated, but this would force to clone/scope to >0.155 that libdef.    
Maybe there will be a codemod pass for this one day.

Reply:
All other plugins had the same syntax so I just followed. Have fixed them now across the definition